celery/app/base.py

```diff
@@ -778,6 +778,10 @@ def send_task(self, name, args=None, kwargs=None, countdown=None,
             **options
         )
 
+        stamped_headers = options.pop('stamped_headers', [])
```

Comment:
do we need additional unit tests for this part? or they are already covered? the Integration tests are good

Reply:
The codecov marks these lines as covered, although I didn't add anything specific (makes sense though). I think the current tests for this PR should be enough, but I'll keep it in mind.